chapters/ch42.md

```diff
@@ -1,35 +1,37 @@
 
 
 
 > If your training and training dev sets include audio recorded within a car, you should also double-check your system’s performance on this subset of data. If it is doing well on the car data in the training set but not on car data in the training dev set, then this further validates the hypothesis that getting more car data would help. This is why we discussed the possibility of including in your training set some data drawn from the same distribution as your dev/test set in the previous chapter. Doing so allows you to compare your performance on the car data in the training set vs. the dev/test set.
 
---> _replace THIS LINE by your translation for the above line_
+Nếu tập huấn luyện và tập huấn luyện-phát triển của bạn chứa những đoạn audio thu trong ô tô, bạn nên kiểm tra kĩ lưỡng chất lượng của hệ thống của bạn trên tập con dữ liệu này. Nếu hệ thống hoạt động tốt với dữ liệu ô tô trong tập huấn luyện nhưng không tốt với dữ liệu ô tô trong tập huấn luyện-phát triển, điều đó càng khẳng định giả thuyết rằng thu thập thêm dữ liệu ô tô sẽ có ích. Đó là lí do tại sao chúng ta thảo luận trong chương trước về khả năng thêm một số dữ liệu lấy từ tập phát triển/tập kiểm tra với cùng phân phối vào trong dữ liệu huấn luyện của bạn. Làm như vậy cho phép bạn so sánh chất lượng [hệ thống] trên tập huấn luyện so với trên tập phát triển/kiểm tra.
 
 
 > Unfortunately, there are no guarantees in this process. For example, if you don't have any way to get more training data that better match the dev set data, you might not have a clear path towards improving performance.
 
---> _replace THIS LINE by your translation for the above line_
+Thật không may, không có một sự bảo đảm nào trong quá trình này. Ví dụ, nếu ban không có cách nào để có thể có thêm dữ liệu huấn luyện tương đồng với dữ liệu trong tập phát triển, bạn có thể không có được một lộ trình rõ ràng nào hướng đến cải thiện chất lượng hệ thống.
 
 
 > **FOOTNOTE:**
 
 > [14] There is also some research on "domain adaptation"—how to train an algorithm on one distribution and have it generalize to a different distribution. These methods are typically applicable only in special types of problems and are much less widely used than the ideas described in this chapter.
 
---> _replace THIS LINE by your translation for the above line_
+**GHI CHÚ**
+
+[14] Cũng có một số nghiên cứu về "thích nghi theo miền" -- làm sao để huấn luyện một giải thuật trên một phân phối và để nó tổng quát hóa trên một phân phối khác. Những phương pháp này thường chỉ ứng dụng được với một số loại vấn đề đặc biệt, và ít được sử dụng hơn nhiều so với những ý tưởng đã được trình bày trong chương này.
```

Comment:
* "domain adaption" đã có trong glossary, nó là "thích ứng miền".
* Từ "algorithm" có thể dịch là "thuật toán" hoặc" giải thuật", nhưng cần nhất quán. Từ đầu sách nó được dịch là "thuật toán" nên mình đề xuất bạn dịch lại là "thuật toán".
* Từ "problem" có nhiều nghĩa, vừa là "vấn đề", vừa là "bài toán". Trong trường hợp này mình nghĩ "bài toán" hợp lý hơn.